app/config/specs/swagger2-latest-server.json

```diff
@@ -28548,72 +28421,86 @@
             "properties": {
                 "key": {
                     "type": "string",
-                    "description": "Parent framework key.",
+                    "description": "Framework key.",
                     "x-example": "sveltekit"
                 },
                 "name": {
                     "type": "string",
                     "description": "Framework Name.",
                     "x-example": "SvelteKit"
                 },
-                "logo": {
-                    "type": "string",
```

Comment:
Any particular reason for removing logo?

Reply:
Was never actually sent from server